script.js

```diff
@@ -0,0 +1,105 @@
+// DOM Elements
+const time = document.getElementById('time');
+const greeting = document.getElementById('greeting');
+const name = document.getElementById('name');
+const focus = document.getElementById('focus');
+const SHOW_AM_PM = true;
+const MIDNIGHT = 0;
+const NOON = 12;
+const ZERO = '0';
+const SIX_HOURS = 6;
+const EIGHTEEN_HOURS = 18;
+const TWENTY_THREE_HOURS = 23;
+
+// Show Time
```

Comment:
The name of the function describes its meaning. So, you don't have to leave a comment.

Reply:
DONE!